src/query/frontend/opencypher/grammar/MemgraphCypher.g4

```diff
@@ -364,13 +366,17 @@ setReplicationRole : SET REPLICATION ROLE TO ( MAIN | REPLICA )
 
 showReplicationRole : SHOW REPLICATION ROLE ;
 
+showReplicationCluster : SHOW REPLICATION CLUSTER ;
+
 replicaName : symbolicName ;
 
 socketAddress : literal ;
 
 registerReplica : REGISTER REPLICA replicaName ( SYNC | ASYNC )
                 TO socketAddress ;
 
+registerMain : REGISTER MAIN TO socketAddress ;
```

Comment:
I think this is wrong.
The coordinator shouldn't care who currently is MAIN/REPLICA, it should only care about the cluster and individual instances.
This ties into another sticky point:
Currently the ASYNC/SYNC replication is dictated by MAIN. REPLICA has no idea if it is ASYNC or SYNC. This needs to change. The REPLICA should be the one saying what communication type it wants. That way, when we switch who main is, we can continue with the same setup as before.

Reply:
Should MAIN be given a name?

Reply:
> The coordinator shouldn't care who currently is MAIN/REPLICA, it should only care about the cluster and individual instances.

Out of scope. But coordinator needs to handle differently main and replicas in any way

Reply:
> Currently the ASYNC/SYNC replication is dictated by MAIN. REPLICA has no idea if it is ASYNC or SYNC. This needs to change. The REPLICA should be the one saying what communication type it wants. That way, when we switch who main is, we can continue with the same setup as before.

Out of scope

Reply:
> Should MAIN be given a name?

Can but no need atm. We will support this in the future